### DISCUSSION TITLE
Beatmap groups (as seen on stable)

### DISCUSSION THREAD

**Discussion by junamat** [ppy/osu#21906] [Ideas] [answered]
As it currently stands, we can only sort beatmaps but cannot get groups as we used to in stable. 

![image](https://user-images.githubusercontent.com/52723130/209869611-aefe9462-e381-406b-b143-441442136c5e.png)

From the old grouping feature, we only have star rating (with the Difficulty range slider) and collections, contained within a dropdown. 
Having this feature back on lazer is relevant cause it makes, for example, searching for ranked sets only more accessible, which right now would have to be done by typing "status=r" on the search box. 

Not too important, but I've seen someone looking for groups so I thought it could be nice to see them back

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/4919

**Reply by junamat** [reply]
I came across this issue before, but I'm a bit confused by this comment: 

> This can already be done either via the filter options in settings (search "song select") or using "diff>6 diff <7" in the search bar. Same as stable.

However, I cannot find any filter-related stuff in settings (not under song select either!) (perhaps an old feature moved somewhere else?) so my original concern hasn't really gone away. Let me rephrase quickly here; Is there any plans to add stable's group by artist; ranked status; title; length; etc as something more accessible than just typing it on the search box?

**Reply by bdach** [reply]
yes there are.

**Reply by junamat** [reply]
ok, thank you.
